src/sentry/static/sentry/app/components/group/inboxBadges/inboxReason.tsx

```diff
@@ -179,7 +179,7 @@ const TooltipWrapper = styled('div')`
 `;
 
 const TooltipDescription = styled('div')`
-  color: ${p => p.theme.gray200};
+  color: ${p => p.theme.subText};
```

Comment:
Will this be right in dark mode?

Reply:
F

Reply:
its fine in dark mode, false alarm.
![image](https://user-images.githubusercontent.com/1400464/108436349-21595080-7200-11eb-9a7d-155610ed60d1.png)